src/sentry/south_migrations/0368_backfill_grouphash_last_processed_event_data.py

```diff
@@ -0,0 +1,1034 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import itertools
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.iterators import chunked
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+        try:
+            from sentry.models import Event, GroupHash
+        except ImportError:
+            # If the model(s) don't exist, we can safely skip the migration.
+            return
+
+        try:
+            record_last_processed_event_id = GroupHash.record_last_processed_event_id
+        except AttributeError:
+            # If the class method doesn't exist, we can safely skip the migration.
+            return
+
+        from sentry.tasks.unmerge import get_fingerprint
```

Comment:
This is a little weird but it's important to keep these two implementations in sync, so I just imported it here.